ujt/transformers.py

```diff
@@ -118,13 +137,11 @@ def apply_change_over_time_classes(
     for sli in slis:
         node_name_sli_map[sli.node_name].append(sli)
 
-    output_elements = []
+    out_elements = []
     for element in elements:
-        ujt_id = element["data"]["ujt_id"]
-        if ujt_id not in node_name_sli_map:
-            output_elements.append(element.copy())
-        else:
-            element_copy = element.copy()
+        new_element = copy.deepcopy(element)
```

Comment:
We lose some performance with this copy operation, but this ensures that we don't mutate the inputs. Also, elements is of type `Dict[str, Union[Dict[str, str], str]]`, so deepcopy shouldn't be too intensive. Also see https://stackoverflow.com/questions/3975376/understanding-dict-copy-shallow-or-deep for why we use copy.deepcopy over dict.copy